examples/detail_info.py

```diff
@@ -0,0 +1,105 @@
+        print("tags: List of tags describing the object (read-only):")
+        for tag in o.tags:
+            print("    " + tag.k + " = " + tag.v)
+
+        print("timestamp: Date when this version has been created (returned as a datetime.datetime) (read-only): " + str(o.timestamp))
+        print("uid: Id of the user that created this version of the object. Only this ID uniquely identifies users. (read-only): " + str(o.uid))
+        print("user: Name of the user that created this version. Be aware that user names can change, so that the same user ID may appear with different names and vice versa. (read-only): " + str(o.user))
+        print("user_is_anonymous(): Check if the user is anonymous. If true, the uid does not uniquely identify a single user but only the group of all anonymous users in general.: " + str(o.user_is_anonymous()))
+        print("version: Version number of the object (read-only): " + str(o.version))
+        print("visible: True if the object is visible (read-only): " + str(o.visible))
+
+    def node(self, n):
+        if self._node_showed == False:
+            self._node_showed = True
+            print("This is a NODE with the following properties:")
+            self.print_obj_info(n)
+            print("location: The geographic coordinates of the node (data type osmium.osm.Location): " + str(n.location))
+            print("location.lat: Latitude  (y coordinate) as floating point number (read-only): " + str(n.location.lat))
+            print("location.lon: Longitude (x coordinate) as floating point number (read-only): " + str(n.location.lon))
+            print("location.lat_without_check(): Latitude  (y coordinate) without checking if the location is valid (datatype float): " + str(n.location.lat_without_check()))
+            print("location.lon_without_check(): Longitude (x coordinate) without checking if the location is valid (datatype float): " + str(n.location.lon_without_check()))
+            print("valid(): Check that the location is a valid WGS84 coordinate, i.e. that it is within the usual bounds (datatype bool): " + str(n.location.valid()))
+            print("x: X coordinate (longitude) as a fixed-point integer: " + str(n.location.x))
+            print("y: Y coordinate (latitude)  as a fixed-point integer: " + str(n.location.y))
+            print("------------------------------------------------------")
+
+    def way(self, w):
+        if self._way_showed == False:
+            self._way_showed = True
+            print("This is a WAY with the following properties:")
+            self.print_obj_info(w)
+
+            print("nodes: Ordered list of nodes (datatype osmium.osm.WayNodeList) (read-only): " + str(w.nodes))
+            for node in w.nodes:
+                print("    " + str(node))
+                # TODO: how to access lat/lon of nodes?
```

Comment:
node is a NodeRef, so see https://docs.osmcode.org/pyosmium/master/ref_osm.html#osmium.osm.NodeRef 

The documentation says so but admittedly not directly, you have to click through 3 redirects.